Supervisor feedback (#80)
chhoumann/p8: report/src/bluetooth_presence_methods/fingerprinting.tex

\section{Fingerprinting}\label{sec:fingerprinting}

> Contrary to trilateration, the concept behind fingerprinting is creating a so-called map that includes measured RSSI values for all beacons in the system for grid-like positions of the area we want to cover.

Reviewer: ```suggestion
Contrary to trilateration, the concept behind fingerprinting is creating a so-called map that includes measured RSSI values for all beacons in the system for grid-like positions of the area that we want to cover.
```

> This map is created prior to the system going live.

Reviewer: ```suggestion
This map is created prior to the system going online.
```

During the first phase, a measuring device is used to measure RSSIs for known positions.
The RSSI measurements and the actual position of the devices, as well as any additional information that can be used in the model, are collected. 
One approach to collecting these measurements is to partition the area of interest into a grid and use the measuring device to collect data for several seconds in each cell of the grid.\cite{improving_indoor_localization}
The measurements are then labelled with the cell coordinate for later classification.

The second phase of fingerprinting involves the computation of a Bluetooth receiver's location based on the map created in the first phase. 
This is done by examining the map, and analyzing how well a signal vector built from received Bluetooth advertisements from the beacons match the measurements in this map.

> \citeauthor{taking_localization_to_the_wild} \cite{taking_localization_to_the_wild} presents results for a fingerprinting approach to show how varying measured RSSI across devices can be mitigated by using RSSI differences between pairs of beacons.

Reviewer: ```suggestion
The study \cite{taking_localization_to_the_wild} demonstrates how the inconsistencies in measured RSSI across devices can be effectively mitigated. This is achieved through a fingerprinting technique that employs the use of RSSI differences between pairs of beacons.
```
